test/src/Provider/AbstractProviderTest.php

```diff
@@ -299,6 +296,9 @@ public function testGetAccessToken()
         $result = $provider->getAccessToken($grant, ['code' => 'mock_authorization_code']);
 
         $this->assertSame($result, $token);
+        $this->assertSame($raw_response['uid'], $token->uid);
+        $this->assertSame($raw_response['access_token'], $token->accessToken);
+        $this->assertSame($raw_response['expires'], $token->expires);
```

Comment:
Maybe these tests shouldn't be included? I added them because I wanted verification that the UID changes were done right.